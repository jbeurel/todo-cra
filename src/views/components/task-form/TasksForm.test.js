import ShallowRenderer from 'react-test-renderer/shallow';
import TaskForm from './TaskForm.component';
import React from 'react';

describe('[TaskForm]', () => {
  it('should display', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<TaskForm/>);

    const tree = renderer.getRenderOutput();
    expect(tree).toMatchSnapshot();
  });
});

