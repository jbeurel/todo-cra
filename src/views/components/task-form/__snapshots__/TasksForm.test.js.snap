// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[TaskForm] should display 1`] = `
<styled.form
  onSubmit={undefined}
>
  <div
    className="field"
  >
    <Field
      component={[Function]}
      name="label"
      placeholder="Task"
      type="text"
      underlineShow={false}
    />
  </div>
  <div
    className="buttons"
  >
    <div
      className="action"
    >
      <RaisedButton
        disabled={false}
        fullWidth={false}
        labelPosition="after"
        primary={false}
        secondary={false}
        type="submit"
      >
        Save
      </RaisedButton>
    </div>
    <div
      className="action"
    >
      <RaisedButton
        disabled={false}
        fullWidth={false}
        labelPosition="after"
        onClick={undefined}
        primary={false}
        secondary={false}
        type="button"
      >
        Cancel
      </RaisedButton>
    </div>
  </div>
  <Divider
    inset={false}
  />
</styled.form>
`;
